hr_timesheet_hours_billed/models/account_analytic_line.py

```diff
@@ -29,7 +29,8 @@ def _inverse_approved(self):
     def create(self, vals_list):
         for vals in vals_list:
             if not vals.get("unit_amount_billed"):
-                vals["unit_amount_billed"] = vals["unit_amount"]
+                unit_amount = vals.get("unit_amount")
+                vals["unit_amount_billed"] = 0.0 if not unit_amount else unit_amount
```

Comment:
You can replace these 2 lines with a single one:
`vals["unit_amount_billed"] = vals.get("unit_amount", 0.0)`